markdown/src/viewer.c

```diff
@@ -337,6 +337,12 @@ markdown_viewer_get_html(MarkdownViewer *self)
     update_internal_text(self, "");
   }
 
+  if((self->priv->text->len > 1) && (self->priv->text->str[0]=='<') ) 
+  {
+	  html = g_malloc(strlen(self->priv->text->str) + 1); 
+	  strcpy(html, self->priv->text->str); html[strlen(self->priv->text->str)]='\0';
+  }
+  else
```

Comment:
Weird code formatting and indentation and stuff. Please follow the same style and formatting as the other code.